Gems/WhiteBox/Code/Source/Viewport/WhiteBoxPolygonTranslationModifier.cpp

```diff
@@ -301,7 +306,7 @@ namespace WhiteBox
             m_polygonView->m_outlines = outlines;
             m_polygonView->m_triangles = triangles;
         }
-
+        m_polygonView->m_transform = AZ::Transform::CreateTranslation(normal * ed_whiteBoxPolygonViewOverlapOffset);
```

Comment:
uber nit: can we please leave the line break after the closing brace please, just helps with separating blocks out